### ISSUE TITLE
UIButton  sd_setImageWithURL: change button frame in UIBarButtonItem CustomView.

### ISSUE THREAD

**Issue by HTOne** [rs/SDWebImage#2053]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by bpoplauschi** [comment]
@HTOne please fill in the info required in the issue template. Also, I don't understand what the problem is, can you detail it a bit?

**Comment by wuxiyang-iOS** [comment]
@HTOne 
Do This :
 [btn sd_setImageWithURL:[NSURL URLWithString:imgUrl] forState:UIControlStateNormal placeholderImage:[[UIImage imageNamed:@"icon-默认头像60-email"]scaledToFitSize:CGSizeMake(30, 30)]];

  UIView *vie = [UIView new];
    vie.frame = CGRectMake(0, 0, 40, 40);
    [vie addSubview:btn];
    UIBarButtonItem *leftBarButton = [[UIBarButtonItem alloc] initWithCustomView: vie];

don't this 
 UIBarButtonItem *leftBarButton = [[UIBarButtonItem alloc] initWithCustomView: btn];

**Comment by bpoplauschi** [comment]
Closing - per last comment.
